pkgs/top-level/all-packages.nix

```diff
@@ -11496,6 +11496,8 @@ in
   sbcl_2_1_2 = callPackage ../development/compilers/sbcl/2.1.2.nix {};
   sbcl = sbcl_2_1_2;
 
+  roswell = callPackage ../development/roswell/default.nix { };
```

Comment:
Please move this package to another location like `pkgs/development/tools/roswell/default.nix` and adjust this line accordingly, `pkgs/development/roswell` is too high up for a single package IMO.